lambdas/service/app.py

```diff
@@ -803,6 +803,14 @@ def validate_order(order: str):
         raise BRE(f'Unknown order `{order}`. Must be one of {supported_orders}')
 
 
+def validate_version(version: str):
```

Comment:
I think I get what the problem is. However, I don't understand why this needs to be a top-level function. It looks like it's only called once. Secondly, you're still relying on the order in which keyword arguments are passed to validate_params and the fact that validate_params evaluates the keyword arguments in the order passed. See if you can figure out a more elegant solution to the ordering problem.